swagger.json

```diff
@@ -1324,7 +1324,7 @@
                     "format": "int32"
                 },
                 "plug_connected_boiler": {
-                    "type": "boolean"
+                    "type": "int32"
```

Comment:
@IOOOTAlan it should. I surely need to integrate a CI test with swagger validator